doc/reference/autoinstall-reference.rst

```diff
@@ -432,52 +435,102 @@ In the new format, the ``primary`` section expects a list of mirrors, which can
 * The special ``country-mirror`` value
 * A mapping with the following keys:
 
-  * ``uri``: The URI of the mirror to use, e.g., ``http://fr.archive.ubuntu.com/ubuntu``.
-  * ``arches``: An optional list of architectures supported by the mirror. By default, this list contains the current CPU architecture.
+  * ``uri`` (Required): The URI of the mirror to use, e.g., ``http://fr.archive.ubuntu.com/ubuntu``.
+  * ``arches`` (Optional): A list of architectures supported by the mirror. By default, this list contains the current CPU architecture.
+
+The URI for the archive mirror does not have to be a country mirror, although it may be the most convenient, and can take the URL of any valid Ubuntu mirror. A list of all registered archive mirrors can be found on `Launchpad <https://launchpad.net/ubuntu/+archivemirrors>`_.
+
+Examples:
+
+.. code-block:: yaml
+
+  # Use custom mirrors for all architectures.
```

Comment:
The comment makes it sound like both CUSTOM_MIRROR_* will be used ; while in practice the latter will be used only if the former fails validation. Can we make that clearer somehow?

suggestion:

```yaml
# Use the first custom mirror that seems to work. Do not restrict to specific architectures.
```

Reply:
Yeah, I like your suggestion but with a minor tweak.

`  # Use the first custom mirror that works. Do not restrict to specific architectures.`